lib/react_on_rails/helper.rb

```diff
@@ -279,7 +279,7 @@ def rails_context(server_side: true)
           # URLs as UTF-8. This situation can occur in browsers that do not encode the
           # entire URL as UTF-8 already, mostly on the Windows platform (IE11 and lower).
           original_url_normalized = request.original_url
-          if original_url_normalized.encoding.to_s == "ASCII-8BIT"
+          if original_url_normalized.encoding == Encoding::BINARY
             original_url_normalized = original_url_normalized.force_encoding("ISO-8859-1").encode("UTF-8")
```

Comment:
@casperisfine should this next line change to use some constants?

Reply:
It definitely could. Note that I opened over 30 PRs like this one, so I really only focused on what would break if we made this change. `force_encoding("ISO-8859-1")` would still work, so I didn't modify it.

But if it was me, yes I'd change it for `force_encoding(Encoding::ISO_8859_1)`.

Reply:
If you can make that change and CI passes, I'll merge!

Reply:
Thanks for the fix!

Reply:
Done.